pkg/dockerfile/generator.go

```diff
@@ -267,3 +312,13 @@ func filterEmpty(list []string) []string {
 	}
 	return filtered
 }
+
+// TODO(hangtwenty): De-duplicate this - the same function is in config.go
```

Comment:
- [ ] `// TODO(hangtwenty): De-duplicate this - the same function is in config.go`